modules/attributes/pages/document-attributes-reference.adoc

```diff
@@ -17,1100 +17,1057 @@ Need to update the compatibility guide with:
 
 Built-in document attributes can be set and unset using an attribute entry.
 
-Automatically Set:: The attribute is set automatically by Asciidoctor and assigned its default value at processing time if you don't set it explicitly in the API, command line, or document.
-Automatically set attributes can be unset with the `!`.
+Set By Default:: The attribute is set automatically by Asciidoctor and assigned a `*default*` value if not otherwise set.
+The default value is indicated in `bold` (e.g., `*skip*`).
+(A built-in attribute which is not set may still have an implied value, but that value won't be accessible using an attribute reference).
 
-Values::
+Values:: Allowable values for a built-in attribute when set.
+Values shown in italic are instructional and indicate a value type (e.g., _empty_, _any_, _number_, etc).
 +
-* _empty_ value: The _empty_ value indicates the attribute doesn't have an explicit value.
+* _empty_ -- An _empty_ value indicates the attribute doesn't have an explicit value.
 The attribute is simply turned on by being set.
-* *default* value: Attributes that have built-in values may have a default value which is indicated by the *bolded value*.
-If you set the attribute but leave the value empty, Asciidoctor will fall back to this default value at processing time.
-* If the attribute doesn't accept an _empty_ value or have a *default* value, than you must explicitly assign the attribute an accepted built-in or user-defined value.
-
-Header Set:: The attribute can be set and assigned a value (or unset) in the document's header.
-In almost all cases, these attributes can also be set, defined, or unset in the API or command line.
-If you set an attribute from the command line or API, it's defined for the whole document and can't be changed in the document header or body unless `@` is added to the end of the value.
-See xref:assignment-precedence.adoc[] for more information.
-The one exception to this rule is the `sectnums` attribute, which can always be changed.
+In some cases, an empty value implies a non-empty value.
+An implied value is prefixed with an equals sign and enclosed in square brackets (e.g., [=`auto`]).
+* _any_ -- Any value is accepted.
+* (`implied`) -- Built-in attributes that are not set may have an implied value.
+The implied value is enclosed in brackets (e.g., (`attributes`)).
```

Comment:
```suggestion
The implied value is enclosed in round brackets (e.g., (`attributes`)).
```

Reply:
I'm curious, is there any difference between round brackets and parentheses? Maybe "parenthesis" can be use symbolically whereas "round bracket" is strictly a character on a keyboard?

Reply:
Good point. We should use parenthesis in this case.